src/store/panesSettings/chart.ts

```diff
@@ -31,6 +31,8 @@ export interface ChartPaneState {
   horizontalGridlinesColor: string
   showVerticalGridlines: boolean
   verticalGridlinesColor: string
+  showWatermark: boolean
+  watermarkColor: string
```

Comment:
typescript definition is good but to enabled vue reactivity on those property at first load, we need to defined default value as well (because undefined = not reactive in the vue component)

the default option for chart pane are defined just below the ts definition in the state constant :

```
const state = {
...
  showWatermark: true,
  watermarkColor: 'rgba(255,255,255,.1)'
```